src/main/java/pro/java/hw19/dao/LessonDao.java

```diff
@@ -0,0 +1,106 @@
+package pro.java.hw19.dao;
+
+import pro.java.hw19.config.DataBaseConnection;
+import pro.java.hw19.dto.Homework;
+import pro.java.hw19.dto.Lesson;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class LessonDao {
+    public void addLesson(Lesson lesson) {
```

Comment:
Better to get connector via constructor only ones instead of check it in each method

Reply:
fixed